Reduced the number of docker images by one `pybamm:latest` (#3992)
pybamm-team/PyBaMM: docs/source/user_guide/installation/install-from-docker.rst

Use the following command to pull the PyBaMM Docker image from Docker Hub:

.. code:: bash

>     docker pull pybamm/pybamm:latest

Reviewer: Ditto
```suggestion
    docker pull pybamm/pybamm
```

1. In your terminal, use the following command to start a Docker container from the pulled image:

.. code:: bash

>     docker run -it pybamm/pybamm:latest

Reviewer: As `:latest` is our only and default tag, user won't need to specify it explicitly for now, hence we can keep it like this - 
```suggestion
    docker run -it pybamm/pybamm
```

- ``IDAKLU``: For IDA solver provided by the SUNDIALS plus KLU.
- ``JAX``: For Jax solver.
- ``ALL``: For all the above solvers.

To build the Docker images with optional arguments, you can follow these steps for each solver:

.. tab:: JAX solver

      .. code-block:: bash

            docker build -t pybamm:jax -f scripts/Dockerfile --build-arg JAX=true .

.. tab:: IDAKLU solver

      .. code-block:: bash

            docker build -t pybamm:idaklu -f scripts/Dockerfile --build-arg IDAKLU=true .

.. tab:: All solvers

      .. code-block:: bash

            docker build -t pybamm:all -f scripts/Dockerfile --build-arg ALL=true .

After building the Docker images with the desired solvers, use the ``docker run`` command followed by the desired image name. For example, to run a container from the image built with all optional solvers:

.. code-block:: bash

      docker run -it pybamm:all

Activate PyBaMM development environment inside docker container using:

.. code-block:: bash

      conda activate pybamm

If you want to exit the Docker container's shell, you can simply type:

> .. code-block:: bash

Reviewer 2: Could we keep these two instructions?

Author: I think we already have those instructions in the above documentation block (`building docker image locally from source`)

Reviewer 2: Oops, missed those. Thanks!

>   PyBaMM Docker image comes with all the solvers by default. These solvers include ``IDAKLU`` IDA solver provided by the SUNDIALS plus KLU and ``JAX`` solver.

Reviewer 2: ```suggestion
  PyBaMM's Docker image comes with all available solvers by default. These solvers include ``IDAKLU`` IDAS solver provided by the SUNDIALS linked with SuiteSparse's KLU and the ``JAX`` solver.
```

A small comment on the wording here. The IDAKLU solver is the IDA solver with sensitivities, hence the extra S ([`IDAS`](https://sundials.readthedocs.io/en/latest/idas/)). SUNDIALS is not shipped with KLU by default and is provided by SuiteSparse.
